graph/api/src/main/scala/org/apache/spark/graph/api/CypherSession.scala

```diff
@@ -88,19 +90,19 @@ object CypherSession {
         }
         .reduce(_ && _)
 
-      NodeFrame(nodes.filter(predicate), ID_COLUMN, labelSet.map(_.substring(1)), nodeProperties)
+      NodeDataset(nodes.filter(predicate), ID_COLUMN, labelSet.map(_.substring(1)), nodeProperties)
     }
   }
 
   /**
-   * Extracts [[RelationshipFrame]]s from a [[Dataset]] using column name conventions.
+   * Extracts [[RelationshipDataset]]s from a [[Dataset]] using column name conventions.
    *
    * For information about naming conventions, see [[CypherSession.createGraph]].
    *
    * @param relationships relationship dataset
    * @since 3.0.0
    */
-  def extractRelationshipFrames(relationships: Dataset[Row]): Set[RelationshipFrame] = {
+  def extractRelationshipDataset(relationships: Dataset[Row]): Set[RelationshipDataset] = {
```

Comment:
Same as above regarding plural form
```suggestion
  def extractRelationshipDatasets(relationships: Dataset[Row]): Set[RelationshipDataset] = {
```